index.html

```diff
@@ -25,13 +25,16 @@
     <header class="hamburger">
       <div class="head">
         <div class="name-me">Yohannes Zemed</div>
-        <img class="mob-img" src="icons/menu.png" alt="menu" />
+        <img id="btn" class="mob-img" src="icons/menu.png" alt="menu" />
         <div class="desk-head">
           <ul class="desk-li">
-            <li>Portfolio</li>
-            <li>About</li>
-            <li>Contacts</li>
-            <li>
+            <li class="close">
+              <img id="close" src="icons/Icon.png" alt="" />
+            </li>
```

Comment:
- [ ] **Required:** You have done an amazing job. However, I have noticed that the close "X" shows up on the desktop view. please fix this.
- [ ] **Optional:** There are some sections such as the header, projects and contact that have overflowing empty spaces that have not been fixed, please see the attached gif for more.

![x overflowing](https://user-images.githubusercontent.com/27709832/150929323-7abe0117-5295-4a67-9dc7-a91fdcb4f149.gif)